frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -294,3 +294,24 @@ Feature: Topology chart area
               And user drag the connector from the deployment workload
               And user drops the connector on the enabled bindable resource
              Then user will see service binding connection
+
+
+        @regression @odc-6361
+        Scenario: Search with label: T-06-TC26
+            Given user has created a deployment workload "nodejs-1"
+              And user has created a deployment workload named "nodejs-2"
```

Comment:
I'm wondering why these two lines use a different step definition, the implementation looks exactly the same.
Please choose one formulation and stick to it for consistency's sake.

Reply:
We have test cases following this approach so I used it, I've made it consistent